charts/operator-wandb/charts/parquet/templates/deployment.yaml

```diff
@@ -113,6 +113,26 @@ spec:
             {{- include "parquet.extraEnv" (dict "global" .Values.global "local" .Values) | nindent 12 }}
             {{- include "wandb.extraEnvFrom" (dict "root" $ "local" .) | nindent 12 }}
 
+          livenessProbe:
+            httpGet:
+              path: /ready 
+              port: 8087
+            initialDelaySeconds: 30
```

Comment:
In future releases we may want to consider parameterizing these values so we can adjust on a per-install basis.  For now this is fine.